### PR TITLE
W-15059913-downgrade-rtf-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/downgrade-runtime-fabric.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
+= Downgrading Runtime Fabric Versions
+
+If you need to downgrade Runtime Fabric version 2.6.x, you must:
+
+. Patch Runtime Fabric to the lower version
```
Comment (digvijaysk19) [inline modules/ROOT/pages/downgrade-runtime-fabric.adoc]: 1. Downgrade runtime fabric.
2. Perform additional steps to switch to daemon set log forwarding. 
(These steps are applicable to customers that use AM log forwarding, other customers do not need these steps).

**Review by digvijaysk19 — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/downgrade-runtime-fabric.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+. <<downgrade-rtf,Downgrade Runtime Fabric version>>
+
+== Downgrade Runtime Fabric Version
+
+To perform the downgrade of version, follow these steps:
```
Comment (digvijaysk19) [inline modules/ROOT/pages/downgrade-runtime-fabric.adoc]:
```suggestion
After downgrading of runtime fabric, follow these steps:
```

File: modules/ROOT/pages/downgrade-runtime-fabric.adoc [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+----
+curl -X PATCH -H 'Authorization: Bearer <bearer-token-here>' -H 'Content-Type: application/json;charset=UTF-8' -H 'Accept: application/json, text/plain, */*' \
+https://anypoint.mulesoft.com/runtimefabric/api/organizations/<orgId>/fabrics/<fabricId>/logforwarding/appScoped?enabled=false
+----
+
+. Optionally, in the event am-log-forwarder daemon-set is not up and running, start the service by using either any of these options:
+
+* In the Anypoint Monitoring UI (*RuntimeFabrics --> Fabric --> LogForwarding*) check the option *Show logs in Anypoint Monitoring*.
+
+* Send an API request:
```
Comment (digvijaysk19) [inline modules/ROOT/pages/downgrade-runtime-fabric.adoc:16]:
```suggestion
* OR Send an API request:
```
There are 2 ways either via UI as stated above or by means of API

Comment (kevintroller) [reply modules/ROOT/pages/downgrade-runtime-fabric.adoc:16]: Yes, that's why I indicated:
"start the service by using either any of these options:"

Comment (digvijaysk19) [reply modules/ROOT/pages/downgrade-runtime-fabric.adoc:16]: Sorry my bad, I misread that.

**Review by digvijaysk19 — DISMISSED** [review]
lgtm

File: modules/ROOT/pages/downgrade-runtime-fabric.adoc [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
++
+----
+curl -X PATCH -H 'Authorization: Bearer <bearer-token-here>' -H 'Content-Type: application/json;charset=UTF-8' -H 'Accept: application/json, text/plain, */*' \
+https://anypoint.mulesoft.com/runtimefabric/api/organizations/<orgId>/fabrics/<fabricId>/logforwarding -d '{"anypointMonitoring": true}'
+----
+
+[NOTE]
+Mule apps redeployed while Runtime Fabric agent version was 2.6.x, must be redeployed to reduce the resource allocation per the downgraded rtf-agent version.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/downgrade-runtime-fabric.adoc]:
```suggestion
Mule apps redeployed while Runtime Fabric agent version was 2.6.x must be redeployed to reduce the resource allocation per the downgraded rtf-agent version.
```

File: modules/ROOT/pages/downgrade-runtime-fabric.adoc [hunk 5]
```diff
@@ -0,0 +1,25 @@
+= Downgrading Runtime Fabric Versions
+
+If you need to downgrade Runtime Fabric version 2.6.x, you must downgrade the version by patching Runtime Fabric to the lower version, and if you use AM log forwarding, additionally perform the following steps:
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/downgrade-runtime-fabric.adoc]:
```suggestion
If you need to downgrade Runtime Fabric version 2.6.x, you must downgrade the version by patching Runtime Fabric to a lower version. If you use AM log forwarding, you must additionally perform the following steps:
```

File: modules/ROOT/pages/downgrade-runtime-fabric.adoc [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+. Enable daemon-set log forwarding by calling the following API (bearer token required):
++
+----
+curl -X PATCH -H 'Authorization: Bearer <bearer-token-here>' -H 'Content-Type: application/json;charset=UTF-8' -H 'Accept: application/json, text/plain, */*' \
+https://anypoint.mulesoft.com/runtimefabric/api/organizations/<orgId>/fabrics/<fabricId>/logforwarding/appScoped?enabled=false
+----
+
+. Optionally, in the event am-log-forwarder daemon-set is not up and running, start the service by using either any of these options:
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/downgrade-runtime-fabric.adoc]:
```suggestion
. Optionally, if am-log-forwarder daemon-set is not up and running, start the service by using either any of these options:
```

File: modules/ROOT/pages/downgrade-runtime-fabric.adoc [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+* In the Anypoint Monitoring UI (*RuntimeFabrics --> Fabric --> LogForwarding*) check the option *Show logs in Anypoint Monitoring*.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/downgrade-runtime-fabric.adoc]:
```suggestion
* In the Anypoint Monitoring UI (*RuntimeFabrics --> Fabric --> LogForwarding*), check *Show logs in Anypoint Monitoring*.
```

**Review by Crispy-Salesforce — DISMISSED** [review]

**Review by luanamulesoft — APPROVED** [review]

**Pull request merged**